File: src/main/java/net/ccbluex/liquidbounce/ui/font/fontmanager/api/FontManager.java
/*
 * FDPClient Hacked Client
 * A free open source mixin-based injection hacked client for Minecraft using Minecraft Forge by LiquidBounce.
 * https://github.com/UnlegitMC/FDPClient/
 */
package net.ccbluex.liquidbounce.ui.font.fontmanager.api;

@FunctionalInterface
public interface FontManager {

	FontFamily fontFamily(FontType fontType);

	default FontRenderer font(FontType fontType, int size) {
		return fontFamily(fontType).ofSize(size);
	}
}

File: src/main/java/net/ccbluex/liquidbounce/ui/font/fontmanager/api/FontManager.kt
/*
 * FDPClient Hacked Client
 * A free open source mixin-based injection hacked client for Minecraft using Minecraft Forge.
 * https://github.com/SkidderMC/FDPClient/
 */
package net.ccbluex.liquidbounce.ui.font.fontmanager.api

fun interface FontManager {
    fun fontFamily(name: String): FontFamily

    fun font(name: String, size: Int): FontRenderer =
        fontFamily(name).ofSize(size)
}
